docs/radio.rst

```diff
@@ -132,6 +133,27 @@ Functions
 
     A ``ValueError`` exception is raised if conversion to string fails.
 
+.. py:function:: receive_all()
```

Comment:
So, my `dict` command tells me:

```
  61 Moby Thesaurus words for "details":
     brass tacks, bric-a-brac, cameo, catalog, cataloging, chaff,
     character, character sketch, characterization, chicken feed,
     chickenshit, delineation, depiction, description, essential facts,
     essentials, evocation, froth, gimcrackery, graphic account, image,
     imagery, impression, itemization, knickknackery, limning, minutiae,
     particularization, peanuts, photograph, picture, portrait,
     portraiture, portrayal, profile, rendering, rendition,
     representation, rubbish, sketch, small beer, small change,
     specification, the data, the details, the dope, the facts,
     the information, the particulars, the picture, the scoop,
     the score, the specifics, the whole story, trash, trifles, trivia,
     trumpery, vignette, vivid description, word painting
```

I've been racking my brain to think of something that's a simple alternative that'll make sense to kids and be easy to type. @deshipu I totally get your point and I think it a good one. Not sure we'll get a better alternative though. Perhaps a good alternative is that it's something we should highlight in the docs (see pending changes landing in a moment).